### PR TITLE
66-introduction-for-maxence-guindon

### LINKED ISSUE #66: Introduction Maxence Guindon
New developer need to update the [AI Lab website](https://ai-cfia.github.io/ai-cfia-ia-acia.github.io/docs/about/team/internships) with their introductions.

### Task:

- [x] Update introduction page in french and english.

### Acceptance criteria:
- The introduction should be professional, reflecting the employee's role within the team and outlining their professional journey.
- It should contains linkedin link and mail.

### PR DESCRIPTION
**Pull request by MaxenceGui**
Add Maxence's introduction in both files:
- internships.md
- the-software-development-team.md

### CODE REVIEW

File: docs/about/team/the-software-development-team.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ sidebar_position: 2
 
 # The Software development team
 
+### [Maxence Guindon](https://www.linkedin.com/in/maxenceguindon)
+
+Hello! My name is Maxence Guindon, and I am a backend developer at Canadian Food Inspection Agency AI-Lab.
+
+I’m working on Nachet. It is a tools that will help inspector to identify seed in an image using an AI model. More specifically, my role in the backend is to make sure that the frontend, the database and the AI model are able to talk to each other. How would you ask? By building efficient API and route for data to travel safely.
```
Comment (rngadam) [inline docs/about/team/the-software-development-team.md]: It is a tool that will help analysts in our science labs to identify seeds

Comment (rngadam) [reply docs/about/team/the-software-development-team.md]: By building efficient API and routes for data to travel safely.

File: docs/about/team/internships.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ sidebar_position: 4
 
 # Internships at the AI Lab
 
+### [Maxence Guindon](https://www.linkedin.com/in/maxenceguindon) - Backend Hiver 2024
+> J'ai choisi de postuler au stage chez l'Agence canadienne d'inspection des aliments parce qu'il s'agissait d'une opportunité de développer mes habiletés et d'apprendre la profession dans un milieu ouvert et accueillant.
+
+> Ce qui est intéressant à l'agence, et plus spécifiquement au laboratoire d'intelligence artificielle, c'est que les solutions développées proposent de vraie amélioration dans le quotidien des employés de l'agence. Je considère que construire des logiciels faisant une différence est une plus-value au stage.
+
+> Dès le départ, notre contribution est demandée. J'ai déjà pu proposer des changements au processus d'intégration. D'avoir rapidement cette confiance de la part de nos superviseurs m'indique qu'ils ont à cœur la réussite de leur stagiaire.
```
Comment (rngadam) [inline docs/about/team/internships.md]: J'ai déjà été en mesure de proposer

**Review by rngadam — CHANGES_REQUESTED** [review]

**Comment by rngadam** [comment]
adding @RussellJimmies as a reviewer as I'm a bit dubious on the language in both English and French.

**Review by rngadam — CHANGES_REQUESTED** [review]
rewrap to 80

**Comment by MaxenceGui** [comment]
> rewrap to 80

Done with rewrap extension (alt+q) could be useful for other intern as well. Thank you @ibrahim-kabir for updating the [doc](https://dev.azure.com/CFIA-DevOps-ACIA/AI-Lab/_wiki/wikis/AI-Lab.wiki/1287/Your-first-pull-request?anchor=creating-an-issue).

File: docs/about/team/internships.md [hunk 3]
```diff
@@ -4,6 +4,22 @@ sidebar_position: 4
...
 # Internships at the AI Lab
 
+### [Maxence Guindon](https://www.linkedin.com/in/maxenceguindon) - Backend Hiver 2024
+> J'ai choisi de postuler au stage chez l'Agence canadienne d'inspection des
+> aliments parce qu'il s'agissait d'une opportunité de développer mes habiletés
+> et d'apprendre la profession dans un milieu ouvert et accueillant.
+
+> Ce qui est intéressant à l'agence, et plus spécifiquement au laboratoire
+> d'intelligence artificielle, c'est que les solutions développées proposent de
+> vraie amélioration dans le quotidien des employés de l'agence. Je considère
```
Comment (RussellJimmies) [inline docs/about/team/internships.md]: [...] proposent de vraie**s** amélioration**s** [...]

File: docs/about/team/the-software-development-team.md [hunk 4]
```diff
@@ -4,6 +4,25 @@ sidebar_position: 2
 
 # The Software development team
 
+### [Maxence Guindon](https://www.linkedin.com/in/maxenceguindon)
+
+Hello! My name is Maxence Guindon, and I am a backend developer at Canadian Food
```
Comment (RussellJimmies) [inline docs/about/team/the-software-development-team.md]: [...] at the Canadian Food Inspection Agency's AI Lab.

File: docs/about/team/the-software-development-team.md [hunk 5]
```diff
@@ -4,6 +4,25 @@ sidebar_position: 2
...
+Inspection Agency AI-Lab.
+
+I’m working on Nachet. It is a tools that will help analysts in our science labs
```
Comment (RussellJimmies) [inline docs/about/team/the-software-development-team.md]: I'm currently working on Nachet, a web application using computer vision AI models, which is designed to help CFIA seed analysts identify seeds from images taken from digital microscopes.

File: docs/about/team/the-software-development-team.md [hunk 6]
```diff
@@ -6,11 +6,20 @@ sidebar_position: 2
...
+to identify seeds in an image using an AI model. More specifically, my role in
+the backend is to make sure that the frontend, the database and the AI model are
+able to talk to each other. How would you ask? By building efficient API and
+routes for data to travel safely.
+
+Before that, I work in escape room, where I was a game master. Working for a
+small company, we had the opportunity to test our skills in different areas and
+propose solution. So during the pandemic, I propose a system to fuse the hint
```
Comment (rngadam) [inline docs/about/team/the-software-development-team.md]: solutions

I proposed

hints

**Review by rngadam — CHANGES_REQUESTED** [review]

File: docs/about/team/the-software-development-team.md [hunk 7]
```diff
@@ -4,6 +4,25 @@ sidebar_position: 2
...
 
+### [Maxence Guindon](https://www.linkedin.com/in/maxenceguindon)
+
+Hello! My name is Maxence Guindon, and I am a backend developer at Canadian Food
+Inspection Agency AI-Lab.
+
+I’m working on Nachet. It is a tools that will help analysts in our science labs
+to identify seeds in an image using an AI model. More specifically, my role in
+the backend is to make sure that the frontend, the database and the AI model are
+able to talk to each other. How would you ask? By building efficient API and
```
Comment (RussellJimmies) [inline docs/about/team/the-software-development-team.md]: "But how? " you might ask. By building efficient APIs and routes [...]

File: docs/about/team/the-software-development-team.md [hunk 8]
```diff
@@ -4,6 +4,25 @@ sidebar_position: 2
...
+routes for data to travel safely.
+
+Before that, I work in escape room, where I was a game master. Working for a
```
Comment (RussellJimmies) [inline docs/about/team/the-software-development-team.md]: [...] I work**ed in** [...]

File: docs/about/team/the-software-development-team.md [hunk 9]
```diff
@@ -4,6 +4,25 @@ sidebar_position: 2
...
+small company, we had the opportunity to test our skills in different areas and
```
Comment (RussellJimmies) [inline docs/about/team/the-software-development-team.md]: I had the opportunity to test my skills in different areas and propose innovative solutions.

File: docs/about/team/the-software-development-team.md [hunk 10]
```diff
@@ -4,6 +4,25 @@ sidebar_position: 2
...
+propose solution. So during the pandemic, I propose a system to fuse the hint
```
Comment (RussellJimmies) [inline docs/about/team/the-software-development-team.md]: [...] I propose**d**

**Review by RussellJimmies — CHANGES_REQUESTED** [review]

File: docs/about/team/internships.md (hunk 3 above)
Comment (MaxenceGui) [reply docs/about/team/internships.md]: Thanks, corrected in next commit

File: docs/about/team/the-software-development-team.md (hunk 4 above)
Comment (MaxenceGui) [reply docs/about/team/the-software-development-team.md]: Done

File: docs/about/team/the-software-development-team.md (hunk 5 above)
Comment (MaxenceGui) [reply docs/about/team/the-software-development-team.md]: Done

File: docs/about/team/the-software-development-team.md (hunk 7 above)
Comment (MaxenceGui) [reply docs/about/team/the-software-development-team.md]: Done

File: docs/about/team/the-software-development-team.md (hunk 8 above)
Comment (MaxenceGui) [reply docs/about/team/the-software-development-team.md]: Done

File: docs/about/team/the-software-development-team.md (hunk 9 above)
Comment (MaxenceGui) [reply docs/about/team/the-software-development-team.md]: Done

File: docs/about/team/the-software-development-team.md (hunk 6 above)
Comment (MaxenceGui) [reply docs/about/team/the-software-development-team.md]: Done

File: docs/about/team/the-software-development-team.md (hunk 10 above)
Comment (MaxenceGui) [reply docs/about/team/the-software-development-team.md]: Done

File: docs/about/team/the-software-development-team.md [hunk 11]
```diff
@@ -4,6 +4,24 @@ sidebar_position: 2
 
 # The Software development team
 
+### [Maxence Guindon](https://www.linkedin.com/in/maxenceguindon)
+
+Hello! My name is Maxence Guindon, and I am a backend developer at the Canadian Food
+Inspection Agency AI-Lab.
+
+I’m currently working on Nachet, a web application using computer vision AI models, which is designed to help CFIA seed analysts identify seeds from images taken from digital microscopes. More specifically, my role in
```
Comment (rngadam) [inline docs/about/team/the-software-development-team.md]: still not wrapped here

**Review by rngadam — CHANGES_REQUESTED** [review]

File: docs/about/team/the-software-development-team.md (hunk 11 above)
Comment (MaxenceGui) [reply docs/about/team/the-software-development-team.md]: Done. Might have unwrapped it accidentaly when I corrected mistake

File: docs/about/team/the-software-development-team.md [hunk 12]
```diff
@@ -4,6 +4,26 @@ sidebar_position: 2
 
 # The Software development team
 
+### [Maxence Guindon](https://www.linkedin.com/in/maxenceguindon)
+
+Hello! My name is Maxence Guindon, and I am a backend developer at the Canadian
+Food Inspection Agency AI-Lab.
```
Comment (rngadam) [inline docs/about/team/the-software-development-team.md]: this document seems to have as many way to name our lab than they are people. can we get some consistency here?

**Review by rngadam — CHANGES_REQUESTED** [review]

File: docs/about/team/the-software-development-team.md (hunk 12 above)
Comment (MaxenceGui) [reply docs/about/team/the-software-development-team.md]: Do you want me to also change how the AI Lab is spelled under other interns?

Comment (rngadam) [reply docs/about/team/the-software-development-team.md]: yes

Comment (MaxenceGui) [reply docs/about/team/the-software-development-team.md]: Done

**Review by rngadam — CHANGES_REQUESTED** [review]
@MaxenceGui your PR has conflicts with the main branch so needs to be rebased.

**Comment by MaxenceGui** [comment]
Rebase done

**Review by rngadam — APPROVED** [review]
ensure that @RussellJimmies validates and closes the open conversations with him before merging.

**Review by RussellJimmies — APPROVED** [review]

**Pull request merged**